pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "pandas>=2.0,<=2.2.2",
     "pybtex",
     "pybtex-apa-style",
-    "robustica>=0.1.3, <=0.1.3",
+    "robustica@git+https://github.com/BahmanTahayori/robustica",
```

Comment:
Thank you @BahmanTahayori It looks like, since we weren't using the extra options that depended on `scikit-learn-extra` your change fixed the issue and all tests are passing. That said, I don't love adding a direct link to a fork of someone's project into a personal repo. I just added a comment to https://github.com/CRG-CNAG/robustica/issues/6 to see if Miquel is interested in making this change more centrally. Otherwise, we might want to consider making a robustica fork into https://github.com/ME-ICA so that more of us will be able to update it directly if future issues arise.

Reply:
Thank you @handwerkerd. I just used the fork to see if all tests passed which was successful. Thanks for contacting Miquel. When he fixes it, I think our issues with `scikit-learn-extra` will be resolved. 

I would like to add a 2d plot for visualisation of the clustering. I think it helps the user to see how runs of components are clustered. I have done it locally and can generate a png file for the result. I believe it is helpful if it is added to the HTML report. Will work on it and will be in touch with you.